index.bs

```diff
@@ -1591,6 +1615,11 @@ To <dfn for="WebTransportSendStream">write</dfn> |chunk| to a {{WebTransportSend
      This sending MUST NOT starve otherwise,
      except for [=flow control=] reasons or [=WritableStream/Error | error=].
 
+     If the sending of |bytes| cannot be completed in its entirety within the current
+     [=flow control=] window, and |atomic| is true, then give up on sending |bytes| and
```

Comment:
```suggestion
     If |atomic| is true and |bytes| cannot all be sent within
     the available [=flow control=] window, then give up on sending |bytes| and
```

Does "give up" mean that the step fails?  That needs to more clear here, I think.

Reply:
How about s/give up on/stop/? The sentence already ends with: _"before proceeding immediately to the next step without failing"_, so it is not failing. This is important to properly clear [[PendingOperation]], which the existing algorithm doesn't do on failure (didn't need to since algo-failure here remains terminal.